package.json

```diff
@@ -5,16 +5,24 @@
   "author": "liamcain",
   "main": "main.js",
   "license": "MIT",
+  "//": "node engine as follows from `process.version` in obsidian; npm for workspace support",
+  "engines": {
+    "node": "^18.18.2",
+    "npm": ">7"
+  },
+  "engineStrict": true,
+  "packageManager": "pnpm@9.3.0+sha512.ee7b93e0c2bd11409c6424f92b866f31d3ea1bef5fbe47d3c7500cdc3c9668833d2e55681ad66df5b640c61fa9dc25d546efa54d76d7f8bf54b13614ac293631",
   "scripts": {
     "lint": "svelte-check && eslint . --ext .ts",
     "build": "npm run lint && rollup -c",
+    "postinstall": "npm run build --workspace=packages/daily-notes-api && npm run build --workspace=packages/ui",
     "test": "jest --passWithNoTests",
     "test:watch": "yarn test -- --watch"
   },
   "dependencies": {
     "obsidian": "obsidianmd/obsidian-api#master",
-    "obsidian-calendar-ui": "0.3.12",
-    "obsidian-daily-notes-interface": "0.9.0",
+    "obsidian-calendar-ui": "*",
+    "obsidian-daily-notes-interface": "*",
```

Comment:
pnpm provides the concept of a `"workspace:^"` pin but as far as I can tell vanilla npm wants you to keep your tracking "normal"; its a matter of style imo